preview-src/ui-model.yml

```diff
@@ -1,9 +1,10 @@
 antoraVersion: 'dev-for-bonita-documentation-theme'
 site:
   keys:
-    nonProduction: false
+    forceDisplaySearchBar: false
```

Comment:
why not only 'displaySearchBar' ?

Reply:
`displaySearchBarInNonProduction` would be probably more accurate but it is longer

Reply:
ok, so in this case I would keep the naming as it is